iacrcc/iacrcc.cls

```diff
@@ -769,11 +769,15 @@
     \checkstring{#2}{error in title}
   \else
      \gdef\@plaintitle{\@IACR@title@plaintext}%
-     \expandafter\checkstring\expandafter{\@IACR@title@plaintext}{plaintext xxargument to title must be plain text}%
+     \expandafter\checkstring\expandafter{\@IACR@title@plaintext}{plaintext argument to title must be plain text}%
   \fi
   \gdef\@title{#2}%
   \if\relax\expandafter\detokenize\expandafter{\@IACR@title@running}\relax
     \gdef\@IACR@title@running{\@plaintitle}%
+    \StrLen{#2}[\IACR@tmp]%
+    \ifnumgreater{\IACR@tmp}{65}{%
+      \ClassError{iacrcc}{The title appears to be very long. Please define a running title by passing running=<runningtitle> to the \string\title\string macro}{}%
```

Comment:
I think this should be ClassWarning and not ClassError. We don't want to risk forcing people to shorten their title if they don't need to. It's possible that a 100 character title will still fit. The title in eprint 2023/1057 is 72 characters but this easily fits.
![image](https://github.com/IACR/latex/assets/14105363/aae4027d-57cd-46ae-b20f-c7730f6f741e)
We should not stop compilation unless we are quite sure that there is something wrong.

Reply:
I disagree. This is also how we handle `\authorrunning`.

Reply:
Changed to Warning